src/components/TimecodeTracker/TimecodeInput.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function TimecodeCard({ title, index, onChange }) {
```

Comment:
The name TimecodeCard has a focus on the ui elment card. Is this naming intentionally?

Reply:
This was a leftover from the first draft. changed it to TimecodeInput. thx